imod/mf6/pkgbase.py

```diff
@@ -8,9 +9,42 @@
 
 import imod
 from imod.mf6.interfaces.ipackagebase import IPackageBase
+from imod.typing.grid import GridDataset
 
 TRANSPORT_PACKAGES = ("adv", "dsp", "ssm", "mst", "ist", "src")
 EXCHANGE_PACKAGES = "gwfgwf"
+ARGS_TO_EXCLUDE = ["validate"]
```

Comment:
I'm not sure this should be here. It might as well be in the derived packages. You could also add an argument to the _get_variable_names method called args_to_exclude

Reply:
Removed!